main.cpp

```diff
@@ -102,6 +102,8 @@ struct Gym
     bool turnOnEquipment(std::string employeeName, bool equipmentState = false);
     // turn on and off electricity
     bool turnOffElectricity(bool switchSetting = true);
+    // new member function
+    Gym findGymLocation(int threshold, int startingValue, Gym &gym);
```

Comment:
References have not been covered yet.  Do not use language features that have not been explained.
References are covered in Project 6 part 2.
You are also returning by copy and the rules that govern copy semantics have not been explained yet.
Also:
a Gym instance will return another Gym?
that doesn't make much sense.
This is a member function, so the only way to call it is by using a Gym instance...
- [ ] revise